Add option to disable domainAllowList in API Gateway (#11907)
kyma-project/kyma: docs/api-gateway/03-02-whitelisted-domains.md

>**TIP:** For more information, read about the [Api CR statuses](#custom-resource-api-rule-status-codes).

By default, the only allowed domain is the domain of the Kyma cluster.

> >**NOTE:** You can disable the allowlist mechanism by changing the value of the `config.enableDomainAllowList` override in the api-gateway component chart.

Reviewer: ```suggestion
To disable the allowlist mechanism, override the value of the **config.enableDomainAllowList** parameter in the API Gateway chart.
```
